Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -64,9 +68,26 @@ def run
         Homebrew.with_no_api_env do
           eval_all = args.eval_all? || Homebrew::EnvConfig.eval_all?
 
-          formulae_and_casks = if args.tap
+          formulae_and_casks = if args.auto?
+            raise UsageError, "`--formula` or `--cask` must be passed with `--auto`." if !args.formula? && !args.cask?
+
+            tap_arg = args.tap
+            raise UsageError, "`--tap=` must be passed with `--auto`." if tap_arg.blank?
+
+            tap = Tap.fetch(tap_arg)
+            autobump_list = tap.autobump
+            what = args.cask? ? "casks" : "formulae"
+            raise UsageError, "No autobumped #{what} found." if autobump_list.blank?
```

Comment:
I feel like this should be a warning instead of a hard error.

Reply:
Not sure -- the non-existence of `autobump.txt` should definitely be an error. An empty `autobump.txt` maybe doesn't need to be, but I don't see why it hurts.

Reply:
@carlocab agreed.